axes_nohw.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import math
+import time
+from numpy import sign
```

Comment:
sign is already defined inline. If we don't need numpy elsewhere, can we leave it out?

Reply:
True. sign and time are unused imports